cherry_picker/cherry_picker.py

```diff
@@ -151,7 +152,9 @@ def set_paused_state(self):
         set_state(WORKFLOW_STATES.BACKPORT_PAUSED)
 
     def remember_previous_branch(self):
-        """Save the current branch into Git config to be able to get back to it later."""
+        """Save the current branch into Git config
+        to be able to get back to it later.
```

Comment:
This seems odd

Reply:
Would this be better?

```suggestion
        """Save the current branch into Git config to be able to get back to it
        later.
```

Reply:
I have an aversion to splitting a one line docstring purely for line-length reasons. Perhaps:

```suggestion
        """Save the current branch into Git config, to be used later."""
```